Remotion/Web/ClientScript/Scripts/ListMenu.ts

```diff
@@ -149,65 +159,71 @@ class ListMenu
 
       if (itemInfo.DiagnosticMetadata)
       {
+        // TODO: merge loops?
         // Do not render empty diagnostic metadata attributes
-        $.each(itemInfo.DiagnosticMetadata, function (key, value)
+        for (const [key, value] of Object.entries(itemInfo.DiagnosticMetadata))
         {
           if (value === "" || value === null)
           {
             delete itemInfo.DiagnosticMetadata[key];
           }
-        });
+        }
 
-        $(item).attr(itemInfo.DiagnosticMetadata);
+        for (const [key, value] of Object.entries(itemInfo.DiagnosticMetadata))
+        {
+          item.setAttribute(key, value as string);
+        }
       }
 
       if (itemInfo.DiagnosticMetadataForCommand)
       {
+        // TODO: merge loops?
         // Do not render empty diagnostic metadata attributes
-        $.each(itemInfo.DiagnosticMetadataForCommand,
-          function (key, value) {
-            if (value === "" || value === null)
-            {
-              delete itemInfo.DiagnosticMetadataForCommand[key];
-            }
-          });
+        for (const [key, value] of Object.entries(itemInfo.DiagnosticMetadataForCommand))
+        {
+          if (value === "" || value === null)
+          {
+            delete itemInfo.DiagnosticMetadataForCommand[key];
+          }
+        }
 
         itemInfo.DiagnosticMetadataForCommand['data-is-disabled'] = isEnabled ? 'false' : 'true';
 
-        $(anchor).attr(itemInfo.DiagnosticMetadataForCommand);
+        for (const [key, value] of Object.entries(itemInfo.DiagnosticMetadataForCommand))
+        {
+          anchor.setAttribute(key, value as string);
+        }
       }
     }
   }
 
-  public static OnKeyDown (event: JQueryKeyEventObject, $listMenu: JQuery): void
+  private static OnKeyDown (event: KeyboardEvent, listMenu: HTMLElement): void
   {
-    ArgumentUtility.CheckNotNullAndTypeIsJQuery ('$tablist', $listMenu);
-
-    var $menuItems = $listMenu.find ('a');
+    var menuItems = listMenu.querySelectorAll('a');
 
     var oldMenuItemIndex = -1;
 
-    var selectedMenuItem = document.activeElement;
+    var selectedMenuItem = document.activeElement as HTMLAnchorElement;
     if (selectedMenuItem != null && TypeUtility.IsDefined (selectedMenuItem.tagName) && selectedMenuItem.tagName.toUpperCase() === 'A')
     {
-      oldMenuItemIndex = $menuItems.index(selectedMenuItem);
+      oldMenuItemIndex = Array.from(menuItems).indexOf(selectedMenuItem);
```

Comment:
We should move the `Array.from` up to the declaration. While this causes a little bit of overhead if the `else` branch is taken, it simplifies future extensions/refactoring.